sonic_installer/main.py

```diff
@@ -576,6 +577,16 @@ def install(url, force, skip_platform_check=False, skip_migration=False, skip_pa
                 "Aborting...", LOG_ERR)
             raise click.Abort()
 
+        # Calling verification script by default - signature will be checked if enabled in bios
```

Comment:
>Calling verification script by default [](http://example.com/codeflow?start=10&length=38)

You mentioned "We want to verify image signature only of secure boot flag is enabled." but the code is verifying image signature always.

Still silently "continue without verification" does not make sense.